### PR TITLE
Fix library style

### PR DESCRIPTION
**Pull request by Givo29**

### CODE REVIEW

File: src/library/entities/Library.java [hunk 1]
```diff
@@ -16,231 +16,225 @@
...
-	
-	public static synchronized Library GeTiNsTaNcE() {		
-		if (SeLf == null) {
-			Path PATH = Paths.get(lIbRaRyFiLe);			
+
+	public static synchronized Library getInstance() {		
+		if (self == null) {
+			Path PATH = Paths.get(libraryFile);			
 			if (Files.exists(PATH)) {	
-				try (ObjectInputStream LiBrArY_FiLe = new ObjectInputStream(new FileInputStream(lIbRaRyFiLe));) {
```
Comment (frankiannelli) [inline src/library/entities/Library.java:56]: check point 9
arguments cannot contain method calls

File: src/library/entities/Library.java [hunk 2]
```diff
@@ -16,231 +16,225 @@
...
 	public static synchronized void SaVe() {
-		if (SeLf != null) {
-			SeLf.lOaN_DaTe = Calendar.gEtInStAnCe().gEt_DaTe();
-			try (ObjectOutputStream LiBrArY_fIlE = new ObjectOutputStream(new FileOutputStream(lIbRaRyFiLe));) {
-				LiBrArY_fIlE.writeObject(SeLf);
-				LiBrArY_fIlE.flush();
-				LiBrArY_fIlE.close();	
+		if (self != null) {
+			self.loanDate = Calendar.getInstance().getDate();
+			try (ObjectOutputStream LibraryFile = new ObjectOutputStream(new FileOutputStream(libraryFile));) {
```
Comment (frankiannelli) [inline src/library/entities/Library.java]: same here

File: src/library/entities/Library.java [hunk 3]
```diff
@@ -16,231 +16,225 @@
...
 		return member;
 	}
 
-	
-	public Book aDd_BoOk(String a, String t, String c) {		
-		Book b = new Book(a, t, c, gEt_NeXt_BoOk_Id());
-		CaTaLoG.put(b.gEtId(), b);		
+
+	public Book addBook(String a, String t, String c) {		
+		Book b = new Book(a, t, c, getNextBookId());
```
Comment (frankiannelli) [inline src/library/entities/Library.java]: same here

**Review by frankiannelli — APPROVED** [review]
Looks good
couple of small things but nothing major

File: src/library/entities/Library.java (hunk 2 above)
Comment (Givo29) [reply src/library/entities/Library.java]: Should be fixed with latest commit

**Pull request merged**